.github/workflows/lint-daggerized.yml

```diff
@@ -0,0 +1,36 @@
+name: lint-daggerized
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+  # Enable manual trigger for easy debugging
+  # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+
+jobs:
+  engine:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: sipsma/dagger-experiments@v5
```

Comment:
for rapid prototyping: you can point to a local path (e.g. `uses: .github/actions/dagger`) and have that code live in this PR until it matures enough to be split it out, if that helps

Reply:
Tried this but I'm not sure if it's working or if something else is bugging out: https://github.com/dagger/dagger/actions/runs/4943782245

Will check again tomorrow. Hopefully it does work because it would be super convenient!